core/tests/protractor_desktop/explorationStatisticsTab.js

```diff
@@ -275,116 +274,119 @@ describe('Statistics tab', function() {
-
-    explorationPlayerPage.submitAnswer('TextInput', 'One');
-    explorationPlayerPage.clickThroughToNextCard();
-    explorationPlayerPage.submitAnswer('TextInput', '2');
-    explorationPlayerPage.viewHint();
-    explorationPlayerPage.submitAnswer('TextInput', '3');
-    explorationPlayerPage.viewSolution();
-    explorationPlayerPage.submitAnswer('TextInput', 'Two');
-    explorationPlayerPage.clickThroughToNextCard();
-    explorationPlayerPage.expectExplorationToNotBeOver();
-    explorationPlayerPage.submitAnswer('TextInput', 'Three');
-    explorationPlayerPage.clickThroughToNextCard();
-    explorationPlayerPage.expectExplorationToBeOver();
-
-    users.logout();
+    await users.login('user2@statisticsTab.com');
+    await libraryPage.get();
+    await libraryPage.findExploration(EXPLORATION_TITLE);
+    await libraryPage.playExploration(EXPLORATION_TITLE);
+
+    await explorationPlayerPage.submitAnswer('TextInput', 'One');
+    await explorationPlayerPage.clickThroughToNextCard();
+    await explorationPlayerPage.submitAnswer('TextInput', '2');
+    await explorationPlayerPage.viewHint();
+    await explorationPlayerPage.submitAnswer('TextInput', '3');
+    await explorationPlayerPage.viewSolution();
+    await explorationPlayerPage.submitAnswer('TextInput', 'Two');
+    await explorationPlayerPage.clickThroughToNextCard();
+    await explorationPlayerPage.expectExplorationToNotBeOver();
+    await explorationPlayerPage.submitAnswer('TextInput', 'Three');
+    await explorationPlayerPage.clickThroughToNextCard();
+    await explorationPlayerPage.expectExplorationToBeOver();
+
+    await users.logout();
 
     // Learner 2 starts the exploration and immediately quits it.
-    users.login('user3@statisticsTab.com');
-    libraryPage.get();
-    libraryPage.findExploration(EXPLORATION_TITLE);
-    libraryPage.playExploration(EXPLORATION_TITLE);
+    await users.login('user3@statisticsTab.com');
+    await libraryPage.get();
+    await libraryPage.findExploration(EXPLORATION_TITLE);
+    await libraryPage.playExploration(EXPLORATION_TITLE);
 
-    explorationPlayerPage.expectExplorationToNotBeOver();
+    await explorationPlayerPage.expectExplorationToNotBeOver();
 
-    users.logout();
+    await users.logout();
 
     // Creator visits the statistics tab.
-    users.login('user1@statisticsTab.com');
-    creatorDashboardPage.get();
-    creatorDashboardPage.navigateToExplorationEditor();
-    explorationEditorPage.navigateToStatsTab();
+    await users.login('user1@statisticsTab.com');
+    await creatorDashboardPage.get();
+    await creatorDashboardPage.navigateToExplorationEditor();
+    await explorationEditorPage.navigateToStatsTab();
 
     // Now, there should be one passerby for this exploration since only learner
     // 3 quit at the first state.
-    explorationStatsTab.expectNumPassersbyToBe('1');
+    await explorationStatsTab.expectNumPassersbyToBe('1');
 
-    users.logout();
+    await users.logout();
   });
```

Comment:
Add check console error in afterEach block

Reply:
replied above.